.github/workflows/formatting-and-tests.yml

```diff
@@ -30,18 +36,29 @@ jobs:
         strategy:
             max-parallel: 4
             matrix:
-                dotnet-version: ["3.1", "5.0", "6.0"]
+                working-directory: ["Flagsmith.Engine","Flagsmith.EngineTest"]
         
         steps:
           - name: Cloning repo
             uses: actions/checkout@v2
             with:
                 fetch-depth: 0
 
-          - name: Set up Dotnet ${{ matrix.dotnet-version }}
+          - name: Set up Dotnet 3.1
+            uses: actions/setup-dotnet@v1
+            with:
+                dotnet-version: 3.1.x
+                
+          - name: Set up Dotnet 5.0
```

Comment:
I'm not sure why we need to set up all version of dotnet here? Surely it only uses the version that we're targetting (which I think is now 6.0.x?).

Reply:
As previously it was installing all 3 frameworks , i left it same assuming that if later we change the target framework,flow won't be running errors. so Should i set on 6.0 only ?

Reply:
Yes, let's setup 6.0 only. We can do that in the other PR though.

Reply:
> @asadabbasfolio3 I have approved this PR now. Ideally, I'd like to squash and merge this PR, however, I appreciate that may be quite painful to manage as your other PR is likely built on top of the commits from this one?
> 
> What I suggest we do is squash and merge this PR then create a new branch from it and cherry pick the commits from your new branch that are outside of this one on top of the new branch. Does that make sense?

**@matthewelwell Yes,make sense. we can go with that.**